src/database.ts

```diff
@@ -28,15 +28,17 @@ export async function setUp(): Promise<void> {
 /**
  * Indicates whether this user is registered in the database
  * @param id The user id to check
- * @returns {Promise<boolean | undefined>} Whether the user is registered
+ * @returns {Promise<string>} Whether the user is registered
  */
-export async function isUserInDB(id: string): Promise<boolean | undefined> {
+export async function isUserInDB(id: string): Promise<string> {
```

Comment:
This name implies a boolean.
It's probably better to rename it to `getUserStatus` (and update the doc)